snmp/datadog_checks/snmp/data/profiles/f5-big-ip.yaml

```diff
@@ -117,6 +117,8 @@ metrics:
         name: sysMultiHostCpuSoftirq
       - OID: 1.3.6.1.4.1.3375.2.1.7.5.2.1.10
         name: sysMultiHostCpuIowait
+      - OID: 1.3.6.1.4.1.3375.2.1.7.5.2.1.11
+        name: sysMultiHostCpuUsageRatio
```

Comment:
Could you replace `511` with `513` in this file below to fix [CI e2e test](https://dev.azure.com/datadoghq/integrations-core/_build/results?buildId=85828&view=logs&jobId=910f82a4-622d-5ace-f877-8b75c045486c&j=910f82a4-622d-5ace-f877-8b75c045486c&t=57642e18-890a-5eef-b695-2b3c448c1261) ? 

https://github.com/DataDog/integrations-core/blob/16b34b371847451f9d38b120b4a806b02c195c03/snmp/tests/test_e2e_core_vs_python.py#L37-L67

Reply:
@AlexandreYang 
Fixed it! https://github.com/DataDog/integrations-core/pull/10924/commits/8cfb12e6241da94feae74e57e9bca15452bc2997

Reply:
@kei6u There are 2 location you need to fix. It seems you fixed for `test_e2e_v3_version_autodetection`, but not for `test_e2e_v3_explicit_version`. Thanks.

Reply:
@AlexandreYang 
Sorry, I missed it.
Fixed it. https://github.com/DataDog/integrations-core/pull/10924/commits/62349145c581c086cfa6bac6243b13602fdd546e